File: DC/2021/05/21/solution.py
l=list(map(int,input().split()))
f=0
while 1:
    try:
        l.append(int(input()))
        f=1 
    except:
        break 
if f: print(*l[::-1]) 
else:
    for i in l[::-1]:
        print(i)

File: DC/2021/05/21/solution.java
import java.util.*;
public class Hello {

    public static void main(String[] args) {
		//Your Code Here
Scanner in=new Scanner(System.in);
String s=in.nextLine();
String ar[]=s.split(" ");
if(ar.length>1)
{
    for(int i=ar.length-1;i>=0;i--)
    System.out.println(ar[i]+" ");
}
else
{
    ArrayList<Integer> al=new ArrayList<>();
    al.add(Integer.parseInt(s));
    while(in.hasNextInt())
    al.add(in.nextInt());
    for(int i=al.size()-1;i>=0;i--)
    System.out.print(al.get(i)+" ");
}
	}
}